plugins/pkgtorrent/server/pkgtorrent-service.conf

```diff
@@ -0,0 +1 @@
+WSGIScriptAliasMatch ^/[^/]+/torrent_service /var/www/cgi-bin/torrent-service.py
```

Comment:
The script is actually installed as /var/www/cgi-bin/pkgtorrent-service.py, so you should either fix it here or the spec file.